cf-deployment.yml

```diff
@@ -949,6 +949,8 @@ instance_groups:
         staging_upload_user: staging_user
         staging_upload_password: "((cc_staging_upload_password))"
         temporary_use_logcache: true
+        logcache:
+          host: log-cache.service.cf.internal
```

Comment:
Is `log-cache.service.cf.internal` available on older versions?

Wondering about what happens during an upgrade if CC starts talking to `log-cache.service.cf.internal` and it isn't available yet OR if CC is still trying to reach log cache on `doppler.service.cf.internal:8080` and it is no longer there. 

If so, does it just mean app metrics aren't available during the upgrade? Are there other effects?

Reply:
@Gerg does that consequence seem okay to you? Is there anything else we can clear up?

Reply:
Yea, that sounds acceptable. I'd definitely include it in the release notes, so people aren't surprised.